src/__tests__/patients/search/NoPatientsExist.test.tsx

```diff
@@ -1,33 +1,18 @@
-import { Icon, Typography, Button } from '@hospitalrun/components'
-import { mount } from 'enzyme'
+import { render, screen } from '@testing-library/react'
 import React from 'react'
 
 import NoPatientsExist from '../../../patients/search/NoPatientsExist'
 
 describe('NoPatientsExist', () => {
-  const setup = () => mount(<NoPatientsExist />)
+  const setup = () => render(<NoPatientsExist />)
 
   it('should render an icon and a button with typography', () => {
-    const wrapper = setup()
+    setup()
 
-    const addNewPatient = wrapper.find(NoPatientsExist)
-    expect(addNewPatient).toHaveLength(1)
+    expect(screen.getByRole('heading', { name: /patients.noPatients/i })).toBeInTheDocument()
+    expect(screen.getByText('patients.noPatients')).toBeInTheDocument()
```

Comment:
These two lines appear to be the checking the same thing... Can you collapse them?
```suggestion
    expect(screen.getByRole('heading', { name: /patients.noPatients/i })).toBeInTheDocument()
```